stable/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -14,6 +14,7 @@ spec:
 {{ include "azuredisk.labels" . | indent 6 }}
         app: csi-azuredisk-controller
     spec:
+      hostNetwork: true
```

Comment:
We should track down why they added this change.

Reply:
It is a SDN issue https://bugzilla.redhat.com/show_bug.cgi?id=1718389

Reply:
Seems to be a requirement for Openshift because we don't block access to cloud metadata IP.